tests/figures/chart/charts_component.test.ts

```diff
@@ -584,6 +585,34 @@ describe("charts", () => {
     ]);
   });
 
+  test("Chart title updates after copy/pasting to another sheet if title is a cell reference", async () => {
```

Comment:
The test should be in the chart_plugin.test.ts file, not this one. You do not test a component here. Also the test name is a bit wrong, we do not update the range on the copy/paste, the title still references the same range.

There also should be a test on duplicate_sheet, there the references to the ranges of the duplcated sheet need to be updated.